### PR TITLE
Login redirection

### PR DESCRIPTION
**Pull request by Njumbi**

### CODE REVIEW

File: app/src/main/java/com/ultratechies/ghala/ui/home/HomeFragment.kt [hunk 1]
```diff
@@ -287,7 +286,7 @@ open class HomeFragment : Fragment() {
         entries.add(
             PieEntry(
                 ordersvsInventory.roundToInt().toFloat(),
-                statsTitles[0 % statsTitles.size]
+              /*  statsTitles[0 % statsTitles.size]*/
```
Comment (Njumbi) [inline app/src/main/java/com/ultratechies/ghala/ui/home/HomeFragment.kt:289]: @malcolmmaima I commented this part of the code, kindly have a look at it

File: app/src/main/java/com/ultratechies/ghala/ui/home/HomeFragment.kt [hunk 2]
```diff
@@ -389,7 +388,7 @@ open class HomeFragment : Fragment() {
         val data: BarData = createChartData(orderData)
         configureChartAppearance()
         prepareChartData(data)
-        setPieChartData(homeStatsResponse)
+     /*   setPieChartData(homeStatsResponse)*/
```
Comment (Njumbi) [inline app/src/main/java/com/ultratechies/ghala/ui/home/HomeFragment.kt:391]: same to this line @malcolmmaima

File: app/src/main/java/com/ultratechies/ghala/ui/home/HomeViewModel.kt [hunk 3]
```diff
@@ -55,7 +55,7 @@ class HomeViewModel @Inject constructor(val userRepository: UserRepository,val h
             val homeStatsResponse = homeStatsRepo.getStats(id)
             when(homeStatsResponse){
                 is APIResource.Success->{
-                    _stats.emit(homeStatsResponse.value)
+            /*        _stats.emit(homeStatsResponse.value)*/
```
Comment (Njumbi) [inline app/src/main/java/com/ultratechies/ghala/ui/home/HomeViewModel.kt:58]: same to this @malcolmmaima

**Review by malcolmmaima — APPROVED** [review]
Everything else looks good ... l'll merge and create a PR to fix the issue you've mentioned

**Pull request merged**